controllers/comments.js

```diff
@@ -7,4 +7,21 @@ exports.getReviewComments = (req, res, next) => {
     }).catch((err) => {
         next(err);
     })
+};
+
+exports.postReviewComment = (req, res, next) => {
+    const { review_id } = req.params;
+    const { username, body } = req.body;
+
+    insertReviewComment(review_id, username, body).then((comment) => {
+        res.status(201).send({ comment });
+    }).catch((err) => {
+        if(err.code === '23503') {
```

Comment:
could these be handled in your error handler? rather than at the bottom of your controller?